### PR TITLE
Move Specific Tests in TinkerTransaction Graph to Tinker Graph

### PR DESCRIPTION
**Pull request by ryn5**

### CODE REVIEW

File: tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java [hunk 1]
```diff
@@ -429,6 +445,7 @@ public void shouldRemoveIndexForRemovedVertex() throws InterruptedException {
      }
 
     @Test
+    @FeatureRequirement(featureClass = Graph.Features.GraphFeatures.class, feature = FEATURE_USER_SUPPLIED_IDS)
```
Comment (Cole-Greer) [inline tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java:448]: Are all of these `@FeatureRequirement` annotations needed in TinkerTransactionGraphTest or are those annotations just for stuff under `gremlin-test`?

File: tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java [hunk 2]
```diff
@@ -31,6 +33,12 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.atomic.AtomicLong;
 
+import static org.apache.tinkerpop.gremlin.structure.Graph.Features.EdgeFeatures.FEATURE_ADD_EDGES;
+import static org.apache.tinkerpop.gremlin.structure.Graph.Features.EdgeFeatures.FEATURE_REMOVE_EDGES;
+import static org.apache.tinkerpop.gremlin.structure.Graph.Features.ElementFeatures.FEATURE_ADD_PROPERTY;
+import static org.apache.tinkerpop.gremlin.structure.Graph.Features.GraphFeatures.FEATURE_TRANSACTIONS;
+import static org.apache.tinkerpop.gremlin.structure.Graph.Features.VertexFeatures.*;
```
Comment (Cole-Greer) [inline tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java:40]: TinkerPop doesn't like wildcard imports, this should likely be replaced with individual imports.

**Review by Cole-Greer — APPROVED** [review]
This looks good to me although gremlin-test isn't an area I understand very well. I would check with someone who knows this better to confirm if `@FeatureRequirement` is needed in the migrated tests and then I think this is ready for an external PR.

File: tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java (hunk 1 above)
Comment (ryn5) [reply tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java:448]: Yes I was asked to add this tag to all tests that have user supplied vertex ids but I'll need Ken to double check whether I hit them all or too many

File: tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java (hunk 2 above)
Comment (ryn5) [reply tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java]: Replaced wildcard import

**Pull request closed without merging**